src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/HoverHandler.cs

```diff
@@ -103,26 +103,28 @@ public async Task<Hover> HandleRequestAsync(TextDocumentPositionParams request,
             if (mappingResult == null || mappingResult.Ranges[0].IsUndefined())
```

Comment:
Do we still need this mapping for C#?

Reply:
We don't map hover locations in the c# server since it doesn't have cross file results, so I believe you still need to map it.